src/app/views/teams/teams-view/TeamList.test.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { render, screen } from "../../../utilities/tests/test-utils";
+import TeamsList from "./TeamsList";
+import "@testing-library/jest-dom/extend-expect";
+
+const teams = {
```

Comment:
I don't think this is the shape of the object you should get as props. we don't utilise `count`